drivers/gps.py

```diff
@@ -81,7 +80,7 @@ def run(self):
                 dt, __, data = packet
                 out = self.process(data)
```

Comment:
This has the same bug Spider had - at most one `publish` call per one call to `listen`.